Storage/rest/advanced_hypermedia.h

```diff
@@ -34,15 +34,21 @@ SOFTWARE.
 #include "hypermedia.h"
 #include "sfinae.h"
 
-CURRENT_STRUCT(AdvancedHypermediaRESTRecordResponse) {
+namespace current {
+namespace storage {
+namespace rest {
+
+CURRENT_STRUCT_T(AdvancedHypermediaRESTRecordResponse) {
+  CURRENT_FIELD(success, Optional<bool>);
```

Comment:
Oh, I see, this type is reused in the container as an item. Isn't the item type a different type than this response type?